src/plot_api/plot_api.js

```diff
@@ -404,6 +404,11 @@ function opaqueSetBackground(gd, bgColor) {
     setBackground(gd, bgColor);
 }
 
+function blendSetBackground(gd, bgColor) {
+    var blend = Color.combine(bgColor, 'white');
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1939/commits/22a598bb814498a2057c2ea665195e77c0a2f598